### PR TITLE
[KYUUBI #3045] Support to do admin rest request with kyuubi-ctl

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

Support to do admin rest request with kyuubi-ctl.

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/CliConfig.scala [hunk 1]
```diff
@@ -21,7 +21,7 @@ import org.apache.kyuubi.ctl.ControlObject.ControlObject
 
 private[ctl] object ControlAction extends Enumeration {
   type ControlAction = Value
-  val CREATE, GET, DELETE, LIST, LOG, SUBMIT = Value
+  val CREATE, GET, DELETE, LIST, LOG, SUBMIT, ADMIN = Value
```
Comment (yaooqinn) [inline kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/CliConfig.scala:24]: admin seems not a verb

Comment (turboFei) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/CliConfig.scala]: thanks, change to administer

**Comment by turboFei** [comment]
@ulysses-you  recommend that introduce a new ctl named kyuubi-adminctl.

I will try to raise another pr.

**Comment by turboFei** [comment]
close this one and in favor of #3057

**Pull request closed without merging**